MAUI/Popup/working-with-mvvm.md

```diff
@@ -0,0 +1,444 @@
+                <Entry Text="{Binding Password}" Grid.Row="1" Grid.Column="1" IsPassword="True"/>
+                <Button Grid.Row="2" Grid.ColumnSpan="2" x:Name="LoginButton" Text="Login" 
+                        Command="{Binding LoginCommand}" HorizontalOptions="Center" />
+            </Grid>      
+            <local:PopupPage x:Name="popup" 
+                             IsOpen="{Binding IsOpen}">                         
+            </local:PopupPage>
+        </StackLayout>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# tabtitle="ViewModel.cs" %}
+public class ViewModel : INotifyPropertyChanged
+{
+    #region Fields
+    
+    private bool isOpen;
+    private string labelString, userName, password;
+    
+    #endregion
+
+    #region Properties
+
+    public ICommand LoginCommand { get; set; }
+
+    public string LabelString
+    {
+        get { return labelString; }
+        set
+        {
+            labelString = value;
+            OnPropertyChanged(nameof(LabelString));
+        }
+    }
+    public string UserName
+    {
+        get { return userName; }
+        set
+        {
+            userName = value;
+            OnPropertyChanged(nameof(UserName));
+        }
+    }
+    public string Password
+    {
+        get { return password; }
+        set
+        {
+            password = value;
+            OnPropertyChanged(nameof(Password));
+        }
+    }
+    public bool IsOpen
+    {
+        get { return isOpen; }
+        set
+        {
+            isOpen = value;
+            OnPropertyChanged(nameof(IsOpen));
+        }
+    }
+
+    #endregion
+
+    public ViewModel()
+    {
+        LoginCommand = new Command(OnLogin);
+    }
+
+    // Executed when the login button is clicked.
+    private void OnLogin()
+    {
+        // You can write your set of codes that needs to be executed.
+        if ((UserName != "Syncfusion") || (Password != "12345"))
+        {
+            // If credentials are incorrect show the error message as popup.
+            IsOpen = true;
+        }
+    }
+
+    #region PropertyChanged
+
+    void OnPropertyChanged(string propertyName)
+    {
+        PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+    }
+
+    public event PropertyChangedEventHandler PropertyChanged;
+    
+    #endregion
+}
+{% endhighlight %}
+{% endtabs %}
+
+## How to retrieve text value from entry control in MAUI popup content template?
+
+This can be done by binding the text property of the entry to a property in the view model class.
+
+Refer to the following code example to retrieve text value from entry control in MAUI popup content template.

```

Comment:
Refer to the following code example to retrieve text value from entry control in the MAUI popup content template.